panamaom/src/PanamaCIRCTOM.scala

```diff
@@ -31,11 +31,17 @@ abstract class PanamaCIRCTOMEvaluatorValue {
   val value: OMEvaluatorValue
 
   // Incomplete. currently for debugging purposes only
-  def display(): String = {
+  override def toString: String = {
     this match {
-      case v: PanamaCIRCTOMEvaluatorValuePath      => s"path{${v.asString}}"
-      case v: PanamaCIRCTOMEvaluatorValueList      => s"[ ${v.elements.map(_.display).mkString(", ")} ]"
-      case v: PanamaCIRCTOMEvaluatorValuePrimitive => s"prim{${v.displayMlirAttr}}"
+      case v: PanamaCIRCTOMEvaluatorValuePath      => s"path{${v.toString}}"
```

Comment:
Don’t need `prim` or extra strings for toString?

Reply:
I prefer keeping these info for debugging purposes.